openapi.yaml

```diff
@@ -18,6 +18,7 @@ tags:
   - name: Listing Search
   - name: Make
   - name: Model
+  - name: Version
```

Comment:
From my understanding, this will only sort tags, nothing else. Now, the question is, do we need it at all?

Reply:
maybe not, tbh I'm not sure